_posts/2023-11-29-java-17.adoc

```diff
@@ -0,0 +1,75 @@
+---
+layout: post
+title: 'Java 17'
+date: 2023-11-29
+tags: announcement
+synopsis: Quarkus 3.7 will move Quarkus to use Java 17 as the default JDK
```

Comment:
I don't like us using `default` because if you're using Java 21, your projects will automatically target Java 21. Also, we might want to make Java 21 the default option on code.quarkus.io very soon.
```suggestion
synopsis: Quarkus 3.7 will move Quarkus to use Java 17 as the minimal JDK
```